File: JAVA/SDLC/DUMMY/Sum.java
package com.ust;

public class Sum {
	public static void main(String args[])
	{
	//an array of numbers
		int[] numbers= {3,4,5,-5,0,12};
		int sum=0;
		
		//iterating through each element of the array
		for(int number :numbers) {
			sum+=number;
		}
		System.out.println("Sum="+sum);
	}
}

		

File: TYPESCRIPT/Sum.ts
function addNumbers(...nums:number[]){
   
    var sum=0;

    for(var i=0;i<nums.length;i++){
        sum=sum+nums[i];
    }
    console.log("sum of the numbers",sum)
}
addNumbers(1,2,3)
addNumbers(10,10,10,10,10,10)